examples/gtest/gtest_minimal.cpp

```diff
@@ -0,0 +1,21 @@
+//  Copyright (c) 2020 ETH Zurich
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#include <gtest/gtest.h>
+
+#include <hpx/hpx_main.hpp>
```

Comment:
Is the sequencing of the headers important (just curious)?

Reply:
I've tried changing the order of the header before thinking of the interface target and it didn't solve my linking problem